app/src/geckoRelease/java/org/mozilla/fenix/engine/GeckoProvider.kt

```diff
@@ -68,28 +63,6 @@ object GeckoProvider {
             runtimeSettings.fontSizeFactor = fontSize
         }
 
-        // Add safebrowsing providers for China
-        if (Config.channel.isMozillaOnline) {
-            val mozcn = SafeBrowsingProvider
-                .withName("mozcn")
-                .version("2.2")
-                .lists("m6eb-phish-shavar", "m6ib-phish-shavar")
-                .updateUrl(CN_UPDATE_URL)
-                .getHashUrl(CN_GET_HASH_URL)
-                .build()
-
-            runtimeSettings.contentBlocking.setSafeBrowsingProviders(mozcn,
-                // Keep the existing configuration
-                ContentBlocking.GOOGLE_SAFE_BROWSING_PROVIDER,
-                ContentBlocking.GOOGLE_LEGACY_SAFE_BROWSING_PROVIDER)
-
-            runtimeSettings.contentBlocking.setSafeBrowsingPhishingTable(
-                "m6eb-phish-shavar",
-                "m6ib-phish-shavar",
-                // Existing configuration
-                "goog-phish-proto")
-        }
```

Comment:
I thought we need to take out this part from beta too?

Reply:
Beta compiles with the missing `import` added 😅